content/docs/ops/traffic-management/troubleshooting/index.md

```diff
@@ -62,6 +62,91 @@ to propagate to all the sidecars.  With large deployments the
 propagation will take longer and there may be a lag time on the
 order of seconds.
 
+## Destination rule policy not activated
+
+Although destination rules are associated with a particular destination host,
+the activation of subset-specific policies depends on route rule evaluation.
+
+The first step in Envoy's service routing process is to evaluate the route rules in
+a virtual service corresponding to the requested host, if there are any,
+to determine if a subset (that is, specific version) of the destination is being routed to.
+If so, the policies corresponding to the subset, if any, are then evaluated.
+Consequently, policies that are defined for specific subsets are only applied if
+the corresponding subset is explicitly routed to.
+
+For example, consider the following destination rule as the one and only configuration defined for the
+*reviews* service, that is, there are no route rules in a corresponding virtual service definition:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: reviews
+spec:
+  host: reviews
+  subsets:
+  - name: v1
+    labels:
+      version: v1
+    trafficPolicy:
+      connectionPool:
+        tcp:
+          maxConnections: 100
+{{< /text >}}
+
+Since there is no specific route rule defined for the *reviews*
+service, default round-robin routing behavior will apply, which will
+presumably call "v1" instances on occasion, maybe even always if "v1" is
+the only running version. Nevertheless, the above policy will never be
+invoked since the default routing is done at a lower level. The rule
+evaluation engine will be unaware of the final destination and therefore
+unable to match the subset policy to the request.
```

Comment:
The phrase "subset policy" is not clear to readers at this point. Do you mean the subset definition in the destination rule, the subset in the routing rule, or the route established by the combination of both?